Makefile

```diff
@@ -18,21 +20,18 @@ GODEP := $(GOPATH)/bin/godep
 $(GODEP):
 	go get -u github.com/tools/godep
 
-build: test
+build:
 	go build -o bin/$(EXECUTABLE) $(PKG)
 
 test: $(PKGS)
 
 $(PKGS): $(GOPATH)/bin/golint
-	@gofmt -w=true $(GOPATH)/src/$@*/**.go
-ifneq ($(NOLINT),1)
-	@echo "LINTING..."
-	@$(GOLINT) $(GOPATH)/src/$@*/**.go
-	@echo ""
-endif
-	@echo "TESTING..."
-	@go test $@ -test.v
-	@echo ""
+	@echo "FORMATTING"
+	@gofmt -l=true $(GOPATH)/src/$@*/*.go
```

Comment:
Neither `golint` nor `go vet` warn if you're failing `gofmt`, it's a basic assumption that code is already compliant.
You should have your editor set up to format on save, let me know if you need help configuring it.